src/core/connection.h

```diff
@@ -563,6 +563,12 @@ typedef struct QUIC_CONNECTION {
     //
     QUIC_LOSS_DETECTION LossDetection;
 
+    //
+    // The total number of packets sent with each corresponding ECT codepoint in all encryption
+    // level.
+    //
+    uint64_t NumPacketsSentWithEct;
```

Comment:
I think this kind of info should be put in `QUIC_SEND`.

Reply:
done